BibBuddy/app/src/main/java/de/bibbuddy/SearchFragment.java

```diff
@@ -234,8 +233,33 @@ private void sortResultList() {
   }
 
   private void handleSearchFilter() {
-    Toast.makeText(context, R.string.filter, Toast.LENGTH_SHORT).show();
-    // TODO filter search results
+    AlertDialog.Builder selectFilterCriteria = new AlertDialog.Builder(context);
+    selectFilterCriteria.setTitle(R.string.filter_to);
+
+    String[] filterChoices = {
```

Comment:
filterCriteria and filterChoices are two complete different words.
Also I named the sortChoices accordingly so I won't change it. You don't need to add String to the name because you see with the datatype that it is a String Array.